### PR TITLE
GRW-1549 / Form template no house

### PR DESCRIPTION
**Pull request by amandaflod**
## Describe your changes

Add NO house template and extrabuildings options. Also added missing fields from SE House. 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1549]: https://hedvig.atlassian.net/browse/GRW-1549?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/services/PriceForm/data/NO_HOUSE.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            ],
+          },
+          layout: { columnSpan: 3 },
+        },
+        {
+          field: {
+            type: 'extra-buildings',
+            name: 'extraBuildings',
+            label: { key: 'Extra buildings' },
+            buildingOptions: [
```
Comment (alebedev) [inline apps/store/src/services/PriceForm/data/NO_HOUSE.ts:119]: Discussion topic - do we need to keep options and their values in sync with the backend?  Are they the same or different per promise, per country?

Not a change for this PR, but let's plan the best way to handle this in the future and leave some notes

File: apps/store/src/components/PriceForm/AutomaticField.tsx [hunk 2]
```diff
@@ -95,6 +95,16 @@ export const AutomaticField = ({ field, onSubmit, loading, autoFocus }: Props) =
       return <SsnSeField field={field} />
 
     case 'extra-buildings':
-      return <ExtraBuildingsField field={field} onSubmit={onSubmit} loading={loading} />
+      return (
+        <ExtraBuildingsField
+          field={field}
+          buildingOptions={field.buildingOptions.map((buildingOption) => ({
+            ...buildingOption,
+            name: translateLabel(buildingOption.label),
```
Comment (alebedev) [inline apps/store/src/components/PriceForm/AutomaticField.tsx:103]: Should we translate here?  Can we do it inside component to keep this case block shorter and ignorant of field types' innards?

Comment (simonauner) [inline apps/store/src/components/PriceForm/AutomaticField.tsx:103]: nit: Could this perhaps overwrite the `label` prop? Then we wouldn't need a new type for `buildingOptions` in ExtraBuildingsField.tsx perhaps, and could just reuse `FieldOption`?

**Review by alebedev — APPROVED** [review]
Well done!  Some nit comments inside

**Review by simonauner — APPROVED** [review]
Nice 🙌 

Got a bit confused by `label`/`name` conversion and the introduction of a new type. Maybe we can reuse `FieldOption`?

File: apps/store/src/components/PriceForm/AutomaticField.tsx (hunk 2 above)
Comment (guilhermespopolin) [reply apps/store/src/components/PriceForm/AutomaticField.tsx:103]: +1 on translating `buildingOptions` in `ExtrBuildingsField`. Building options are included in `field` and `ExtrBuildingsField` component already make usage of `translateLabel` so it feels like a small effort with some gains

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/services/PriceForm/data/NO_HOUSE.ts (hunk 1 above)
Comment (amandaflod) [reply apps/store/src/services/PriceForm/data/NO_HOUSE.ts:119]: I think that is the plan! They will be different depending on promises and countries. Anything to add @robinandeer ?

File: apps/store/src/components/PriceForm/AutomaticField.tsx (hunk 2 above)
Comment (amandaflod) [reply apps/store/src/components/PriceForm/AutomaticField.tsx:103]: That makes sense, I will fix that :)

Comment (amandaflod) [reply apps/store/src/components/PriceForm/AutomaticField.tsx:103]: Ok! I will fix that, will also add that change to the "select" type :) Thanks!

**Pull request merged**